ui/frontend/selectors/index.ts

```diff
@@ -102,7 +103,7 @@ export const miriVersionDetailsText = createSelector([getMiri], versionDetails);
 
 const editionSelector = (state: State) => state.configuration.edition;
 
-export const isWasmAvailable = (state: State) => (
+export const isNightlyChannel = (state: State) => (
   state.configuration.channel === Channel.Nightly
 );
```

Comment:
I'd prefer to have more semantic selectors, so something like:

```suggestion
const isNightlyChannel = (state: State) => (
  state.configuration.channel === Channel.Nightly
);
export const isWasmAvailable = isNightlyChannel;
export const isHirAvailable = isNightlyChannel;
```

Reply:
Sure thing, done.